app/assets/javascripts/application.js

```diff
@@ -10,7 +10,18 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
+
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
+//= require bootstrap-sprockets
+//= require filepicker
+//= require js-routes
 //= require_tree .
+
+
+function openHelpTab() {
+  url = 'https://github.com/osc/frame-renderer'
+  var win = window.open(url, '_blank');
+  win.focus();
+}
```

Comment:
Instead of using JavaScript to open a help page, a simpler and more appropriate way is to just use HTML directly: `<a href="https://github.com/osc/frame-renderer" target="_blank">Help</a>`

Reply:
Done, yea, not quite sure why I did that. But yea, just a link now.